tests/confreq-mathml-rs-behavior/OPS/package.opf

```diff
@@ -4,7 +4,7 @@
     <dc:title>Title</dc:title>
     <dc:creator>Creator</dc:creator>
     <dc:date>2021-01-01</dc:date>
-    <dc:identifier id="pub-id">NOID</dc:identifier>
+    <dc:identifier id="pub-id">confreq-mathml-rs-behavior</dc:identifier>
```

Comment:
Why is this change in this PR? Seems like it should be in #29.

Reply:
That is a side-effect of my own testing of the script. With the merge of #29 this is now done.